### ISSUE TITLE
FineTuneConfiguration is not passing lrPolicyDecayRate

### ISSUE THREAD

**Issue by papperwing** [deeplearning4j/deeplearning4j#3981]
I was looking into FineTuneConfiguration and found out then in methods which pass parameters into NeuralNetworkConfiguration lrPolicyDecayRate is not passed. It leads into problem when one is trying to set learningRatePolicy except POLY and SCHEDULE.

https://github.com/deeplearning4j/deeplearning4j/blob/master/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/transferlearning/FineTuneConfiguration.java

DL4J tested: 0.9.1
also I didn't found it in sources in 0.9.2-SNAPSHOT

**Comment by eraly** [comment]
There are major improvements/changes coming to learning rate and schedules in general. The learning rate (and schedules) will be passed in via updaters. This antiquates this issue therefore closing.

Here is the PR:
https://github.com/deeplearning4j/deeplearning4j/issues/3981
